src/code/fv-components/bottom-hud.js

```diff
@@ -0,0 +1,20 @@
+import React, {PropTypes} from 'react';
+import TrialCountView from './trial-count';
+
+const BottomHUDView = ({id, className, trial, trialCount}) => {
+
+  return (
```

Comment:
I suppose this is worth a component because the HUD will be expanding a lot, right?